src/routes/jira/events/jira-events-router.ts

```diff
@@ -3,14 +3,24 @@ import { JiraEventsInstallPost } from "routes/jira/events/jira-events-install-po
 import { extractInstallationFromJiraCallback } from "~/src/jira/extract-installation-from-jira-callback";
 import { JiraEventsUninstallPost } from "routes/jira/events/jira-events-uninstall-post";
 import { verifyAsymmetricJwtTokenMiddleware } from "~/src/jira/util/jwt";
+import { saveConfiguredAppProperties } from "utils/app-properties-utils";
 
 export const JiraEventsRouter = Router();
 
 // TODO: remove enabled and disabled events once the descriptor is updated in marketplace
 JiraEventsRouter.post("/disabled", (_: Request, res: Response) => {
 	return res.sendStatus(204);
 });
-JiraEventsRouter.post("/enabled", (_: Request, res: Response) => {
+JiraEventsRouter.post("/enabled", async (req: Request, res: Response) => {
```

Comment:
Seems the enabled is triggered multiple times. https://developer.atlassian.com/cloud/jira/platform/connect-app-descriptor/#lifecycle-attribute-example

![image](https://user-images.githubusercontent.com/105693507/206319368-4297eeeb-5c9a-45f0-ad8b-c07b20e05b2b.png)

Reply:
Then it will "reset" the config status even after the config properties is set to true.

Reply:
We can make another API call to simply check if the app property is already set or not. If its already set, do nothing, and if its not set, then we set it to this default `false`.

Reply:
Sounds good.